docs/integrations/builtin/core-nodes/n8n-nodes-base.scheduletrigger/common-issues.md

```diff
@@ -38,3 +38,7 @@ To set the timezone for an individual workflow:
 1. Select **Settings**.
 1. Change the **Timezone** setting.
 1. Select **Save**.
+
+### Variables not working as expected
+
+While variables can be used in the scheduled trigger, their values only get evaluated when the workflow is activated. After activating the worfklow, you can alter a variable's value in the settings but it won't change how often the workflow runs. To work around this, you must stop and then re-activate the workflow to apply the updated variable value.
```

Comment:
Some of the queries I have seen have been along the lines of "If i have a schedule of 1 hour and I edit the workflow  and save it, does the one hour reset to  from the time I saved it?" It would be good if we could answer that here too!

Reply:
Yes, as a separate entry. Sorry, i didn't mean to extend the scope of this but it just reminded me when we were here

Reply:
No problem. I'll sort that for you.

Reply:
Thanks so much!